src/NServiceBus.RabbitMQ/DelayedDelivery/DelayInfrastructure.cs

```diff
@@ -79,16 +78,16 @@ public static StartupCheckResult CheckForInvalidSettings(SettingsHolder settings
         {
             var routingTopologySupportsDelayedDelivery = settings.GetOrDefault<bool>(SettingsKeys.RoutingTopologySupportsDelayedDelivery);
             var TimeoutManagerDisabled = settings.GetOrDefault<bool>(SettingsKeys.DisableTimeoutManager);
-            var timeoutManagerFeatureActive = settings.GetOrDefault<FeatureState>(typeof(TimeoutManager).FullName) == FeatureState.Active;
+            var externalTimeoutManagerAddress = settings.GetOrDefault<string>("NServiceBus.ExternalTimeoutManagerAddress") != null;
```

Comment:
We used a class with consts for headers to know where those strings are utilized. Should this be included?